.github/workflows/rotki_ci.yml

```diff
@@ -176,6 +176,12 @@ jobs:
           key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
           restore-keys: |
             ${{ runner.os }}-node-
+      - uses: actions/cache@v1
+        with:
+          path: ~/.cache/.rotkehlchen-test-dir
```

Comment:
Just a question is that directory also used in OSX or does OSX uses `~/Library/Caches` instead

Reply:
It's used in OSX too

Reply:
@kelsos if it's also used in OSX this will work, right?

Reply:
Yes, I read the description in the issue after asking the question. You seem to put it in ~/.cache in OSX too so I don't see why it won't work

Reply:
thanks!